server/game/cards/15-W2D/ReggieMallard.js

```diff
@@ -0,0 +1,28 @@
+const DudeCard = require('../../dudecard.js');
+
+class ReggieMallard extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: this,
+            condition: () => this.game.shootout && this.isParticipating(),
+            effect: ability.effects.dynamicBullets(() => this.getHucksterDudesInPosse())
+        });
+    }
+
+    getHucksterDudesInPosse() {
+        if(!this.game.shootout) {
+            return 0;
+        }
+        const Posse = this.game.shootout.getPosseByPlayer(this.controller);
+
+        if(Posse) {
+            // Check if there are more than 4 hucksters in the posse, and if so, return 4, otherwise return the number of hucksters in the posse
+            return (Posse.getDudes(dude => dude.hasKeyword('huckster')).length > 4) ? 4 : Posse.getDudes(dude => dude.hasKeyword('huckster')).length;
```

Comment:
```suggestion
            const hucksters = Posse.getDudes(dude => dude.hasKeyword('huckster'));
            return (hucksters.length > 4) ? 4 : hucksters.length;
```
Let define hucksters first so it is not processed 2x.